src/AppInitializer.tsx

```diff
@@ -96,7 +99,7 @@ function AppInitializer(props: { children?: React.ReactNode }) {
   const [isFramed, setIsFramed] = useState(false)
   const [appId, setAppId] = useState<string>()
   const [redirectURL, setRedirectURL] = useState<string>()
-  const [themeOptions, setThemeOptions] = useState<ThemeOptions>()
+  const [themeOptions, setThemeOptions] = useState<DeprecatedThemeOptions>()
```

Comment:
ThemeOptions have has changed in v5. They provide Deprecated for compatibility. Might be able to fix it in more elegant way